backend/api/views/main.py

```diff
@@ -8,15 +8,13 @@
 
 main = Blueprint("main", __name__)  # initialize blueprint
 
-"""
-function that is called when you visit /rate
+'''
+function that is called when you visit /response_rates
 Parameters
-    year: year string with format YYYY
+    year: year string with format YY
     optional tract_id: 11-digit tract id string
     optional state: two digit id string
-"""
-
-
+'''
```

Comment:
Can you change this back to what it is on master?

Reply:
fixed